### DISCUSSION TITLE
Postgres query

### DISCUSSION THREAD

**Discussion by MrCrypticxDev** [planetscale/discussion#153] [FAQs] [answered]
Curious as to if you guys have any plans on supporting postgres in the future.

**Comment by jgreet** [comment, marked as answer]
@MrCrypticXDev that's a totally understandable thing to ask for, and you can find Postgres fans at PlanetScale, but we have no plans to host PostgreSQL databases.

For more see https://twitter.com/rbranson/status/1461031251691773953
